godot/components/cooking/Applicance.gd

```diff
@@ -3,19 +3,18 @@ extends Node
 # We're leaving some potential for the appliances to have larger capacity (max 4),
 # perhaps through upgrades. Keeping things simple for now with capacities of 1.
 # Think of the state as a kitchen burner with 4 quadrants.
-var type: int
+@export var type: Enums.Appliance
```

Comment:
This is probably overkill since the appliance name isn't really important. The important part is exposing the types of recipes this appliance can make. When the appliance's scene is created, then you can name it "Stockpot" for example

Reply:
yeah, there is a RecipeUtils file where there is a dictionary keeping track of all the recipes each appliance can make, and that depends on the appliance type, so that's where the type comes from. we can change it so that the valid recipes is inherent to the appliance, i.e. `@export var recipes: Array[Enums.Recipe]` if that's what you mean, but i think this works too